packages-ui/roosterjs-react/lib/ribbon/plugin/createRibbonPlugin.ts

```diff
@@ -49,6 +50,15 @@ class RibbonPluginImpl implements RibbonPlugin {
      * @param event PluginEvent object
      */
     onPluginEvent(event: PluginEvent) {
+        if (
+            event.eventType == PluginEventType.KeyDown &&
+            event.rawEvent.key == 'k' &&
+            event.rawEvent.ctrlKey
+        ) {
+            this.onButtonClick(insertLink, 'insertLinkTitle', undefined);
```

Comment:
Suggest we check if insertLink button is added to ribbon. It is possible that user doesn't want this feature.

Or we can also add a parameter to allow disable this feature.

Reply:
Allowing users to disable this feature would be a good to add. Do you suggest a specific way to implement this?

Reply:
So, check if insertLink button is added to ribbon could you elaborate on this a bit. Thank you :)

Reply:
I guess if there is a setting option/button with a dropdown menu to allow users to adjust their preferences about shortcuts, instead of adding a dropdown menu on inertLinkButton directly. This would make much sense, and by default, we can disable this feature and enable it on a switch button from false to true if the user wants to use this feature, do you think?